client/my-sites/controller.js

```diff
@@ -462,48 +462,62 @@ export function siteSelection( context, next ) {
 		return next();
 	}
 
-	const siteId = getSiteId( getState(), siteFragment );
```

Comment:
The block is chained after the resolving of `requestSites` for unlinked checkout flow